### PR TITLE
🧹 Visual housekeeping

### PR DESCRIPTION
**Pull request by henryboldi**
This PR only fixes the account switcher spacing and wallet tab width so far. Small stuff.

⚠️ I've got a few other issues in mind to include fixes for, but still, it's mergeable now.

width
.

### CODE REVIEW

**Comment by Shadowfiend** [comment]
@henryboldi dealers choice here--would you rather accumulate a few more or land this now? Don't really mind either way on my end.

**Comment by henryboldi** [comment]
I'd rather accumulate a few more! That said, if we want to bundle up the testing edition very soon, it might make sense to include now

**Comment by Shadowfiend** [comment]
Cool cool, I'll plan on merging here in an hour or so.

File: ui/pages/Wallet.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default function Wallet(): ReactElement {
   }
 
   return (
-    <div className="wrap">
```
Comment (Shadowfiend) [inline ui/pages/Wallet.tsx:42]: Probably want to nuke the `wrap` class below as well.

Comment (Shadowfiend) [reply ui/pages/Wallet.tsx:42]: Actually peeking more closely, every other page has a `main` element up here (with no further class). We should stick to that pattern here.

**Review by Shadowfiend — APPROVED** [review]
Fixes de tings.

File: ui/pages/Wallet.tsx (hunk 1 above)
Comment (henryboldi) [reply ui/pages/Wallet.tsx:42]: > Actually peeking more closely, every other page has a main element up here (with no further class). We should stick to that pattern here.

They do?

**Pull request merged**